dashboards-observability/.cypress/integration/2_notebooks.spec.js

```diff
@@ -51,7 +51,7 @@ describe('Adding sample data and visualization', () => {
       .should('exist');
     cy.wait(100);
     cy.get('.euiButton__text').contains('Yes').trigger('mouseover').click();
-    cy.wait(100 * 5);
+    cy.wait(100);
```

Comment:
Let's not change tests that were not updated with code. This delay may be introduced earlier as github actions on the functional test repo take longer.

Reply:
Sure. Reverted this change in [1e6978c](https://github.com/opensearch-project/dashboards-observability/pull/1308/commits/1e6978c45b041cd0bf544e0b86f6a5265b30e60a)